data/ABRS_2023/data.csv

```diff
@@ -61518,7 +61516,7 @@ Flora_of_Australia,Cinnamomum baileyanum,leaf_length,lamina,50,,mm,minimum
 Flora_of_Australia,Cinnamomum camphora,leaf_length,lamina,40,,mm,minimum
 Flora_of_Australia,Cinnamomum laubatii,leaf_length,lamina,80,,mm,minimum
 Flora_of_Australia,Cinnamomum oliveri,leaf_length,lamina,85,,mm,minimum
-Flora_of_Australia,Cinnamomum propinquum,leaf_length,lamina,5,,mm,minimum
```

Comment:
Mainly I've curious with how you caught this? By comparing to the same taxon in other studies?

Reply:
Yes, I got the average plant height from all species and then compared them with maximum leaf length - nearly all of the smallest leaves on tall trees were in a few families or genuses and these ones stood out